src/node/markdown/plugins/githubAlerts.ts

```diff
@@ -0,0 +1,75 @@
+import type MarkdownIt from 'markdown-it'
+import type { ContainerOptions } from './containers'
+
+export const gitHubAlertsPlugin = (
+  md: MarkdownIt,
+  options?: ContainerOptions
+) => {
+  const markers = [
+    'TIP',
+    'NOTE',
+    'INFO',
+    'IMPORTANT',
+    'WARNING',
+    'CAUTION',
+    'DANGER'
+  ]
+  const matchCaseSensitive = true
```

Comment:
Any specific reason for having this true? GitHub's implementation seems to be case insensitive.

Reply:
I think it was case-sensitive, was copied from `markdown-it-github-alerts` https://github.com/antfu/markdown-it-github-alerts/commit/34f264da25042a40e50e0475bf1f5c9d392f339e

Let me update this